docs/pages/file-structure.page.server.mdx

```diff
@@ -10,26 +10,26 @@ while for basic apps we recommend a single `pages/` directory.
 
 ```bash
 # Landing page (URL: `/`)
-/pages/index/index.page.js
+/pages/index/+Page.js
 /pages/index/SomeComponentForLandingPage.js
 /pages/index/**/* # More files specific to the landing page
 # About page (URL: `/about`)
-/pages/about/index.page.js
+/pages/about/+Page.js
 /pages/about/SomeComponentForAboutPage.js
 /pages/about/**/* # More files specific to the about page
-# Other pages (we recommend creating a directory for each page, e.g.
-# `/pages/about/index.page.js` instead of `/pages/about.page.js`)
-/pages/*/*.page.js
+# Other pages
+/pages/*/+Page.js
+# Special pages
+/pages/_error/+Page.js
 
 # Components shared by several pages
 /components/*
 
 # Code that specifies how our pages are rendered
-/renderer/_default.page.server.js
-/renderer/_default.page.client.js
-/renderer/_error.page.js
+/renderer/+onRenderHtml.js
+/renderer/+onRenderClient.js
 /renderer/PageShell.js  # The root (React/Vue/...) component that wraps the `Page` component
-/renderer/PageShell.css # exported by `/pages/**/*.page.js`.
+/renderer/PageShell.css # exported by `/pages/**/+Page.js`.
```

Comment:
@brillout did you mean "imported"? Not sure I understand this comment

Reply:
Indeed, it doesn't make sense 😅. I'm 👍 for removing ~~# exported by /pages/**/+Page.js~~.

Reply:
done :) (removed `# exported by /pages/**/+Page.js`)